src/spikeinterface/core/node_pipeline.py

```diff
@@ -43,23 +43,27 @@
 
 class PipelineNode:
     def __init__(
-        self, recording: BaseRecording, return_output: bool = True, parents: Optional[List[Type["PipelineNode"]]] = None
+        self,
+        recording: BaseRecording,
+        return_output: bool | tuple[bool] = True,
+        parents: Optional[list[Type["PipelineNode"]]] = None,
     ):
         """
-        This is a generic object that will make some computation on peaks given a buffer of traces.
-        Typically used for exctrating features (amplitudes, localization, ...)
+                This is a generic object that will make some computation on peaks given a buffer of traces.
+                Typically used for exctrating features (amplitudes, localization, ...)
```

Comment:
@zm711 is this extra indentation on purpose? Looks weird to me!

Reply:
fixed ;)

Reply:
Thanks. It wasn’t intentional. Pre-commit did that to me. I have no clue why…. But thanks!